src/app/products/products.component.spec.ts

```diff
@@ -22,4 +23,20 @@ describe('ProductsComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it("does products name match to its price",()=>{
+    const [first,second]=component.products;
```

Comment:
hmm how does this test make sense?

Reply:
different products have different price .if shoes price is 8000 then if i assign any different price value it shows error

Reply:
first index should contain the first hard coded name and its value

Reply:
So you shouldn't write tests for configurations. Configurations can change. Your products can change. It shouldn't matter. Write tests to ensure your code works the way you want it to. :)